### PR TITLE
Fix wrong timeout param to wget

### LINKED ISSUE #20561: e2e: Services should work after restarting... are causing builds to time out
cc @k8s-oncall

### PR DESCRIPTION
**Pull request by thockin**
Fixes #20561, I hope

@saad-ali if this passes, we should prioritize getting it in.

### CODE REVIEW

**Comment by ihmccreery** [comment]
In this case `-T` and `-connect-timeout` is equivalent:

```
-T seconds
       --timeout=seconds
           Set the network timeout to seconds seconds.  This is equivalent to specifying --dns-timeout, --connect-timeout, and --read-timeout, all at the same time.
```

LGTM.

**Comment by ihmccreery** [comment]
The `--tries` here is the important part, I think.

**Comment by thockin** [comment]
ACK, should I revert the timeout flag?

**Comment by ihmccreery** [comment]
> ACK, should I revert the timeout flag?

As it stands, this should be strictly better, but I do think `-T` is more appropriate.

**Comment by ihmccreery** [comment]
@k8s-oncall please consider merging this manually to get the `-serial` jobs green again.

**Comment by saad-ali** [comment]
> @k8s-oncall please consider merging this manually to get the -serial jobs green again.

Will do once presubmit checks are green

**Comment by thockin** [comment]
Damn, wget in busybox doesn't support timeout < 1 sec.  Update coming.

**Comment by thockin** [comment]
or --tries, sigh

**Comment by thockin** [comment]
Hopefully this will run.

**Comment by thockin** [comment]
third time's a charm

**Comment by bprashanth** [comment]
@k8s-bot test this github issue: #20402

**Comment by thockin** [comment]
A variable shadowing bug in the test may be the culprit for this.  Testing AGAIN.

File: test/e2e/service.go [hunk 1]
```diff
@@ -239,8 +239,10 @@ var _ = Describe("Services", func() {
 		ns := f.Namespace.Name
 		numPods, servicePort := 3, 80
 
+		By("creating service1 in namespace " + ns)
```
Comment (ihmccreery) [inline test/e2e/service.go]: Pull "service1" out into a var?

Comment (ihmccreery) [inline test/e2e/service.go]: Meh, actually, I guess that'd just be `service1 := "service1"`.  Ignore.

File: test/e2e/service.go [hunk 2]
```diff
@@ -251,28 +253,35 @@ var _ = Describe("Services", func() {
...
-			Failf("VIPs conflict: %v", svc2IP)
+			Failf("service IPs conflict: %v", svc2IP)
 		}
 
+		By("verifying service2 is still up")
 		expectNoError(verifyServeHostnameServiceUp(c, ns, host, podNames2, svc2IP, servicePort))
-		expectNoError(verifyServeHostnameServiceUp(c, ns, host, podNames3, svc3IP, servicePort))
 
-		expectNoError(stopServeHostnameService(c, ns, "service2"))
-		expectNoError(stopServeHostnameService(c, ns, "service3"))
```
Comment (ihmccreery) [inline test/e2e/service.go]: Why are you nixing these two lines?  Please update the title of the PR?

**Comment by ihmccreery** [comment]
LGTM except for last comment.

File: test/e2e/service.go (hunk 2 above)
Comment (thockin) [inline test/e2e/service.go]: There's no point in manually removing the services - namespace deletion handles it.  We should, in general, remove cleanup code where it is not related to the test, and depend on the framework.

**Comment by saad-ali** [comment]
Merging manually.

**Pull request merged**